pkg/pipes/pipes.go

```diff
@@ -0,0 +1,182 @@
+package pipes
+
+import (
+	"flappy/pkg/bird"
+	"flappy/pkg/window"
+	"fmt"
+	"github.com/veandco/go-sdl2/img"
+	"github.com/veandco/go-sdl2/sdl"
+	"math/rand"
+	"sync"
+	"time"
+)
+
+const (
+	pipeTexturePath = "../resources/Pipe-3.png"
```

Comment:
⚠️  If you generates a bin from the game this resource maybe could not be resolved.